pkg/api/v1/router_instance_metadata_lookup_test.go

```diff
@@ -70,4 +74,41 @@ func TestGetMetadataLookupByIP(t *testing.T) {
 			assert.Equal(t, testcase.expectedStatus, w.Code)
 		})
 	}
+
+	// Verify cache TTL settings are honored by setting the TTL to 1 second, sleeping to surpass that, and trying again
+	viper.Set("cache_ttl", time.Second)
+	time.Sleep(time.Second)
```

Comment:
```suggestion
	viper.Set("cache_ttl", -time.Second)
```

I believe this should work, and then we don't need to sleep.

Reply:
Cool idea. I've updated this and added a comment explaining why.